packages/web/src/Components/Table/TableRow.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FunctionComponent, forwardRef } from 'react';
+import MaterialTableRow, {
+  TableRowProps as MaterialTableRowProps,
+} from '@material-ui/core/TableRow';
+import { withTheme, makeStyles } from '@material-ui/core/styles';
+import { IThemeWeb } from '../../Themes';
+import { getDefaultTheme } from '../shared';
+
+export interface ITableRowProps extends MaterialTableRowProps {
+  theme?: IThemeWeb | unknown;
+}
+
+export const TableRow: FunctionComponent<ITableRowProps> = forwardRef((
+  props: ITableRowProps,
+  ref: any,
+) => {
+  const { 
+    theme: providerTheme
+  } = props;
+
+  const theme: any  = React.useMemo(() => getDefaultTheme(providerTheme), [providerTheme]);
+  const useStyles = React.useMemo(() => makeStyles({
+    selected: {
+      backgroundColor: `${theme.palette.primary.main}29 !important`
```

Comment:
O que seria este número `29` após o token de cor?
Remover o uso de `!important` por não ser boa prática de _styling_.

Reply:
O número 29 é referente a propriedade alpha do hexadecimal.

De acordo com a cor da UX

https://xd.adobe.com/view/17e8a75e-c241-4d51-9009-265fed2b5188-42bb/screen/5a5d6877-a321-4376-8e4f-38de4f0e868d/

Reply:
O important é devido a um override sobre a propriedade `&:nth-of-type(even)` dentro do Table Component, você tem alguma sugesão de implemetar esse override de estilo?